azure/packages/azure-client/src/AzureFunctionTokenProvider.ts

```diff
@@ -13,32 +13,62 @@ import { AzureMember } from "./interfaces";
  * Azure Fluid Relay token resolution.
  *
  * @deprecated 1.2.0, This API will be removed in 2.0.0
- * No replacement since it is not expected anyone will use this token provider as is
- * See https://github.com/microsoft/FluidFramework/issues/13693 for context
+ * @remarks No replacement since it is not expected anyone will use this token provider as is.
```

Comment:
Note: the `@remarks` tag applies to the class, not the `@deprecated` comment, so I don't think we want to change the tags here. Same with the `@see` - that creates a "See also" section for the class, and will separate the link from the deprecation comment.